### ISSUE TITLE
Copy embed link messaging is confusing for screen reader users

### ISSUE THREAD

**Issue by Paul-Clue** [LLK/scratch-www#7315]
### Where To Find Bug
`/ explore`

### Expected Behaviour
The screen reader should announce the proper labelling attribute attached to the language select element.

### Actual Behaviour
The screen reader does not announce the `aria-label` attribute for the language select element. The screen reader says: "English, required collapsed, pop up button" when focus lands on the element.

### Steps To Reproduce

- Open https://scratch.mit.edu/ 
- Turn-on your screen reader of choice
- Click on the `/explore` page tab
- Tab (or arrow) through the page until focus lands on the language select element in the footer
- Notice that the screen reader says "English, required collapsed, pop up button"

### Screenshot
<img width="1476" alt="Screenshot 2022-12-19 at 11 31 55 PM" src="https://user-images.githubusercontent.com/67766160/208584483-e16aadd5-5f4d-416f-8b94-f7b8bf3b86eb.png">

### Fix
Create an HTML `<label>` element for the language select element.

### System Details
macOs Ventura,Firefox, VoiceOver (AT)

**Comment by benjiwheeler** [comment]
Closing this because the title and description seem mismatched, and it's not clear what the issue refers to.
